policy/triage-bugzilla.md

```diff
@@ -59,6 +59,33 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
 
 An explanation of the release status field.
 
+### Regressions
+
+_A regression is a code change that degrades existing functionality or performance for the user._
+
+#### Is a regression
+
+* Facebook homepage does not render
+* Image elements do not appear in the DOM tree
+* CSS Grid rules are not recognized
+
+#### Is not a regression
+
+* A website dependent on an unimplented part of the CSS specification does not render as intended
+* Firefox implements a feature differently than the user expects it to
+* The reporter is requesting a new feature
+
```

Comment:
A bug that is not dependent on a code change in Firefox but from external factors (e.g. a new crash caused by a Windows update)

Reply:
Adding w/link to Third-party Software Affecting Firefox